lib/workers/branch/auto-replace.spec.ts

```diff
@@ -154,7 +166,6 @@ describe('workers/branch/auto-replace', () => {
       upgrade.depIndex = 0;
       upgrade.replaceString =
         'image: "1111111111.dkr.ecr.us-east-1.amazonaws.com/my-repository:1"\n\n';
-      upgrade.packageFile = 'k8s/base/defaults.yaml';
```

Comment:
Why this change?

Reply:
I have returned this, `outputFile` function was mocked a bit incorrectly